gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationFragment.java

```diff
@@ -261,12 +267,12 @@ private void onShowPhotoSelectorDialog(Field field) {
     addPhotoBottomSheetBinding.setViewModel(viewModel);
     addPhotoBottomSheetBinding.setField(field);
 
-    BottomSheetDialog bottomSheetDialog = new BottomSheetDialog(getContext());
+    BottomSheetDialog bottomSheetDialog = new BottomSheetDialog(requireContext());
     bottomSheetDialog.setContentView(addPhotoBottomSheetBinding.getRoot());
     bottomSheetDialog.setCancelable(true);
     bottomSheetDialog.show();
 
-    AddPhotoDialogAdapter.ItemClickListener listener =
+    Consumer<Integer> consumer =
```

Comment:
I think you can declare this as a private class method, for example `onSelectPhotoClick(type)`...